cmd/root.go

```diff
@@ -173,7 +173,8 @@ func getDataDir(cmd *cobra.Command, logger logger.Logger) string {
 	dataDir, _ = filepath.Abs(filepath.Clean(dataDir))
 
 	if !util.Exists(dataDir) {
-		logger.Fatal("data directory %s does not exist. please create the directory and retry again.", dataDir)
+		os.Mkdir(dataDir, 0700)
```

Comment:
oops... one more ... use os.MkdirAll in case there are multiple directory segments that need to be created.